app-text/ghostwriter/ghostwriter-1.5.0.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="A cross-platform, aesthetic, distraction-free Markdown editor"
+DESCRIPTION="ghostwriter is a text editor for Markdown."
+HOMEPAGE="https://wereturtle.github.io/ghostwriter/"
+COPYRIGHT="wereturtle et al."
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/wereturtle/ghostwriter/archive/v1.5.0.tar.gz"
+CHECKSUM_SHA256="bfd64024d9741f55d587afa5661f4b5e2a48becbbc92495c2186fca44fb08486"
+PATCHES="ghostwriter-$portVersion.patch"
+ADDITIONAL_FILES="ghostwriter.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	ghostwriter$secondaryArchSuffix = $portVersion
+	app:Ghostwriter$secondaryArchSuffix = $portVersion
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	cmd:hunspell$secondaryArchSuffix
+	lib:libQt5Concurrent$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5PrintSupport$secondaryArchSuffix
+	lib:libQt5WebKit$secondaryArchSuffix
+	lib:libQt5WebKitWidgets$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	hunspell${secondaryArchSuffix}_devel
+	devel:libQt5Concurrent$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	devel:libQt5WebKit$secondaryArchSuffix
+	devel:libQt5WebKitWidgets$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:g++$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:lrelease$secondaryArchSuffix >= 5
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:qmake$secondaryArchSuffix >= 5
+	"
+
+BUILD()
+{
+	lrelease ghostwriter.pro
+	qmake
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
+	mkdir -p $dataDir/ghostwriter/translations
+	cp ./build/release/ghostwriter $appsDir/Ghostwriter
+	cp ./translations/*.qm $dataDir/ghostwriter/translations/
+	
+	local APP_SIGNATURE="application/x-vnd.ghostwriter"
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	local LONG_INFO="$SUMMARY"
+	sed \
+		-e "s|@APP_SIGNATURE@|$APP_SIGNATURE|" \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		-e "s|@LONG_INFO@|$LONG_INFO|" \
+		$portDir/additional-files/ghostwriter.rdef.in > ghostwriter.rdef
+	
+	addResourcesToBinaries ghostwriter.rdef $appsDir/Ghostwriter
+	mimeset -f $appsDir/Ghostwriter
+	addAppDeskbarSymlink $appsDir/Ghostwriter
+}
```

Comment:
Please add a newline character (0x0A) at the end if this line.
You can do that with vi:
```
vi ghostwriter-1.5.0.recipe
:w
:q
```